chef/cookbooks/crowbar/templates/default/crowbar-pre-upgrade.sh.erb

```diff
@@ -26,3 +26,8 @@ service pacemaker stop
 echo "No HA setup found..."
 
 <% end %>
+
+for bridge in $(ovs-vsctl --real list-br); do
```

Comment:
It should be enough (an probably safer) if we just do this for bridges which have ports connected to external devices (usually br-public and br-fixed). And only if the ml2 ovs driver is used. Probably easiest to figure out the bridge names in crowbar and set them template variables.